src/isomorphic/classic/class/__tests__/ReactClass-test.js

```diff
@@ -381,4 +381,30 @@ describe('ReactClass-spec', function() {
     );
   });
 
+  it('should remove invalid prop types', function() {
+    var warn = console.error;
+    console.error = jest.genMockFn();
```

Comment:
Don't try catch this. Use spyOn (we do this a bunch and already do it inside the proptypes test)